### PR TITLE
initial commit of library & code

### PR DESCRIPTION
**Pull request by ladyada**
these uart modules may be easier for some situations where you cant measure precise pulses

### CODE REVIEW

File: adafruit_us100.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        for the sensor to handle. In my experience, the sensor can detect
+        objects over 460 cm away.
+        :return: Distance in centimeters.
+        :rtype: float
+        """
+        self._uart.write(bytes([0x55]))
+        data = self._uart.read(2)  # 2 bytes return for distance
+        if len(data) != 2:
+            raise RuntimeError("Did not receive distance response")
+        dist = (data[1] + (data[0] << 8)) / 10
```
Comment (caternuson) [inline adafruit_us100.py:68]: Don't know sensor details, but should this be an `|`?

**Review by caternuson — COMMENTED** [review]
Property names and units check with sensor design guide. Travis is just complaining about TODO's. Just a question in review.

Also - is this something in the shop? do you want any hardware testing done?

File: adafruit_us100.py (hunk 1 above)
Comment (ladyada) [reply adafruit_us100.py:68]: + and | are the same when dealing with shifted bytes :)

**Comment by ladyada** [comment]
no testing needed yet - not in shop, so no URLs yet - coming soon!

**Pull request merged**